Networking/Services/UserManager.swift

```diff
@@ -166,7 +165,7 @@ public extension SessionManager {
     
```

Comment:
since we have to add the authentication headers now we can add a property:

`
var authenticationHeader: [String: String] {
        return ["Authorization": _sessionToken.value]
}
`

Reply:
we could, but as I was thinking of erasing this class completely it didn't make sense... The final objective is that those kind of things are kept per application and not in the framework. Or at least not in this framework

Reply:
Makes sense, but I still think we could use some convenient way of adding authorization headers without having to create the dictionary

Reply:
The way I see it, you will have something like `LoginRepository` that does the login and gets a token, stores it somewhere in a singleton and then you create all your other repositories using  [Authorization: lalala] as default headers. Or you could add that function to the singleton. 
I mean, you still need to have something like SessionManager, the point was to decouple it from the framework